alibi_detect/cd/fet.py

```diff
@@ -0,0 +1,285 @@
+import logging
+import numpy as np
+from typing import Callable, Dict, Tuple, Optional, Union
+from alibi_detect.cd.base import BaseUnivariateDrift
+from scipy.stats import hypergeom
+import numba as nb
+import math
+
+logger = logging.getLogger(__name__)
+
+
+class FETDrift(BaseUnivariateDrift):
+    def __init__(
+            self,
+            x_ref: Union[np.ndarray, list],
+            p_val: float = .05,
+            preprocess_x_ref: bool = True,
+            update_x_ref: Optional[Dict[str, int]] = None,
+            preprocess_fn: Optional[Callable] = None,
+            correction: str = 'bonferroni',
+            alternative: str = 'decrease',
+            n_features: Optional[int] = None,
+            input_shape: Optional[tuple] = None,
+            data_type: Optional[str] = None
+    ) -> None:
+        """
+        Fisher exact test (FET) data drift detector, which tests for a change in the mean of binary data.
+        For multivariate data, the Bonferroni or False Discovery Rate (FDR) correction is applied.
```

Comment:
I know it's implied but lets be explicit that for multivariate its applied to each feature separately.